README.md

```diff
@@ -35,11 +42,13 @@ evervault.api_key = <YOUR-API-KEY>
 result = evervault.encrypt_and_run(<CAGE-NAME>, { 'hello': 'World!' })
 ```
 
-## API Reference
+## Reference
+
+At present, there are two functions available in the Python SDK: `evervault.encrypt()`, `evervault.run()`, and `evervault.encryptAndRun()`.
```

Comment:
```suggestion
```

This proves my point exactly, it has more than two methods, which we missed, and additionally also has `cage_list`

Reply:
what should be added to the `cage_list` section?

Reply:
added the section but marked as TODO

Reply:
It's more or less the same functionality as the ruby one.

```
Evervault.cage_list
Return a CageList object, containing a list of your team's Cages
```